src/containers/my-cooperations/accept-cooperation-close/AcceptCooperationClosing.tsx

```diff
@@ -38,7 +92,32 @@ const AcceptCooperationClosing: React.FC<AcceptCooperationClosureProps> = ({
       }
       icon={<ErrorOutlineRounded />}
       title={t('titles.acceptCooperationClosing')}
-    />
+    >
+      {isInputShown ? (
+        <Box sx={styles.inputBox}>
+          <Typography sx={styles.textGray}>
+            {t('cooperationDetailsPage.InputFieldLabel')}{' '}
+          </Typography>
+          <Box sx={styles.inputField}>
+            <InputField
+              error={!!errors.declineReason}
```

Comment:
You can do it even better, extract `errors.declineReason` to separate variable `hasErrors` and use it in all places, where you need to check if error is present

```suggestion
              error={Boolean(errors.declineReason)}
```